docs/spec/lightclient/verification/Lightclient_A_1.tla

```diff
@@ -29,8 +29,28 @@ VARIABLES
   lightBlockStatus,   (* a function from heights to block statuses *)
   latestVerified      (* the latest verified block *)
 
-(* the variables of the lite client *)  
-lcvars == <<state, nextHeight, fetchedLightBlocks, lightBlockStatus, latestVerified>>  
+(* the variables of the lite client *)
+lcvars == <<state, nextHeight, fetchedLightBlocks, lightBlockStatus, latestVerified>>
+
+(* The light client history, which is the function mapping states (0 .. nprobes) to the record with fields:
+   - verified: the latest verified block in this state
+   - current: the block that is being checked in this state
+   - now: the time point in this state
+   - verdict: the light client verdict for the `current` block in this state
+*)
+VARIABLE
+  history
```

Comment:
Just a question for my learning, I don't see the `history` function actually getting used anywhere except on https://github.com/informalsystems/tendermint-rs/pull/546/files#diff-04b72e322e067c19d149fbabcfe5a830R272 where is is `UNCHANGED`. Could you maybe point out to me where we actually apply the function to make use of the records in its codomain?

Reply:
It's used here: 
 - [LightTests.tla](https://github.com/informalsystems/tendermint-rs/blob/master/light-client/tests/support/model_based/LightTests.tla) to describe the TLA+ tests, 
 - and here [apalache-tendermint.json](https://github.com/informalsystems/tendermint-rs/blob/master/testgen/jsonatr-lib/apalache-tendermint.json) to extract the information from the Apalache counterexample.

It was part of another PR, which is already merged.

Reply:
Also, as we discussed with @josef-widder and @konnov, the way I introduce the history variable, needs to be changed, because this feature is not needed for all users of the spec. So we need to figure out how to modularize the spec, and to allow to add the history as an extension.

Reply:
Ah, got it. Thanks very much for the pointers!